model/raw/src/main/scala/aqua/raw/value/ValueRaw.scala

```diff
@@ -159,21 +159,40 @@ object LiteralRaw {
   }
 }
 
+case class StreamRaw(values: List[ValueRaw], streamName: String, streamType: StreamType) extends ValueRaw {
```

Comment:
Add a comment on why `StreamRaw` has `streamName` while other collections do not